SYSTEMS.md

```diff
@@ -0,0 +1,117 @@
+# System-specific information
+
+This framework strives to be as system-independent as possible, but there are some platform-specific details that you may need be aware of when running these benchmarks.
+Below we collect some information you may want to keep in mind on the different systems.
+
+## ARCHER2
+
+### Home partition
+
+ARCHER2 uses the standard Cray setup for which the home partition is ***not*** mounted on compute nodes, read [Data management and transfer](https://docs.archer2.ac.uk/user-guide/data/) for more details.
+You likely want to install this benchmarking framework outside of your home directory, for example inside `/work/<project code>/<project code>/${USER}`, where `<project code>` is your project code.
+
+### Queue options
+
+When submitting jobs to compute nodes, you need to specify the job queue and maybe the project account.
+The former is specified by setting the [Quality of Service](https://docs.archer2.ac.uk/user-guide/scheduler/#quality-of-service-qos), the latter is necessary if your user account is associated to multiple projects on ARCHER2 and you need to [specify which one to use](https://docs.archer2.ac.uk/user-guide/scheduler/#specifying-resources-in-job-scripts) for the submitted jobs.
+We cannot automatically set these options for you because they are user-specific, but when you run a benchmark you can use the `-J`/`--job-option` flag to `reframe` to add new job options.
+Some examples:
+
+```
+reframe -c examples/sombrero -r --performance-report --system archer2 -J'--qos=serial'
+reframe -c examples/sombrero -r --performance-report --system archer2 -J'--qos=short' -J'--account=t01'
+```
+
+Note that jobs run on specific queues, like `serial` and `lowpriority`, do not charge the corresponding project, but you need to have at least 1 [Computing Resource (CU)](https://docs.archer2.ac.uk/user-guide/scheduler/#cus) in budget to access them.
```

Comment:
Do we want to have this written down in our documentation? Perhaps I'd just rather link to the Archer2 docs.

Reply:
Fair enough
```suggestion
```